### DISCUSSION TITLE
Escaping a period to avoid a list

### DISCUSSION THREAD

**Discussion by andrew-culver** [retypeapp/retype#513] [Q&A] [answered]
markdownguide.org says:

>  If you need to start an unordered list item with a number followed by a period, you can use a backslash (\) to [escape](https://www.markdownguide.org/basic-syntax#escaping-characters) the period.

I want to start a paragraph with the single word "I" (first person pronoun).

If I use:

> I.

I get a list.

If I use a backslash before the period, I get what I want, but there are spurious characters at the top of the page above all other output:

> \n
\n\n" /> \n
\n\n"> \n
\n\n">

**Comment by geoffreymcgill** [comment]
Here's a **sample.md** I'm testing with:

```md
# Sample

1\. Paragraph 1

2\. Paragraph 2
```

and here is the result I'm seeing in the source:

<img width="752" alt="Screenshot 2023-03-17 at 7 53 06 AM" src="https://user-images.githubusercontent.com/62210/225924626-e9d04c70-c4e0-48e3-a498-26701ed37556.png">

Here's the raw generated HTML:

```html
<doc-anchor-target id="sample" class="break-words">
    <h1>
        <doc-anchor-trigger class="header-anchor-trigger" to="#sample">#</doc-anchor-trigger>
        <span>Sample</span>
    </h1>
</doc-anchor-target>
<p>1. Paragraph 1</p>
<p>2. Paragraph 2</p>
```

How can I reproduce the scenario where you are seeing the `\n` newlines?

**Comment by andrew-culver** [comment]
Try replacing the digit "1" with the capital letter "I"

**Reply by geoffreymcgill** [reply]
confirmed. boom.

Looks like it's an issue with the value of some meta tag generation.

Now using the `.md` sample of:

```md
l\. Paragraph 1
```

Here's a sample of one of the generated meta tags:

```html
<meta name="description" content="<ol type=\"a\" start=\"12\">\n<li>Paragraph 1</li>\n</ol>\n" />
```

The HTML generated in the main content is correct:

```html
<p>l. Paragraph 1</p>
```

We are investigating and will fix this immediately. Thanks for reporting the issue.

**Reply by andrew-culver** [reply]
Serves me right for having the temerity to start a paragraph with the single word "I."

Your code was offended.

Which BTW makes it far more powerful than ChatPGT -- which claims to have no opinions.

**Reply by geoffreymcgill** [reply]
You can temporarily work around this issue by adding a `description` to the page. The following sample demonstrates:

```md
---
description: "Add a custom description here"
---
# Sample

l\. Paragraph 1
```

The meta tags will use your custom `description` and the whacked-out newline debris will be gone.

We will be fixing the defect, so as of the v3.0 release, you will be able to remove the `description` config from the page.

**Reply by andrew-culver** [reply]
Good one. Thanks.

**Comment by geoffreymcgill** [comment, marked as answer]
This defect has been fixed and will be included in the upcoming Retype v3.0 release.

Thanks again for reporting the issue. 👍
